src/ESLint.Formatter/sarif.js

```diff
@@ -54,13 +54,20 @@ module.exports = function (results, data) {
 
     const sarifFiles = {};
     const sarifArtifactIndices = {};
-    var nextArtifactIndex = 0;
+    let nextArtifactIndex = 0;
```

Comment:
>let [](start = 4, length = 3)

`let` is preferable because its block scoping behavior is more understandable to most people than `var`'s function body scoping.